lib/riax_vnode.ex

```diff
@@ -16,7 +16,7 @@ defmodule Riax.VNode do
   end
 
   def handle_command({:put, {k, v}}, _sender, state = %{data: data}) do
-    Logger.info("PUT Key: #{k}, Value: #{v}", state)
+    Logger.info("PUT Key: #{inspect k}, Value: #{inspect v}", state)
```

Comment:
these sound more like debug than info to me

Reply:
Changed!